### PR TITLE
There's no reason not to support macOS :)

### PR DESCRIPTION
**Pull request by pawlowskialex**

### CODE REVIEW

**Comment by spxrogers** [comment]
you're absolutely right :)

File: Moya-Gloss.podspec [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "Moya-Gloss"
-  s.version          = "2.0.0-beta.2"
+  s.version          = "2.0.0-beta.3"
```
Comment (spxrogers) [inline Moya-Gloss.podspec]: can you keep the version number the same? 
I'm in the middle of managing new version updates and I'll handle that little bit :)

**Comment by pawlowskialex** [comment]
Sure, done.

**Comment by spxrogers** [comment]
awesome! thank you @pawlowskialex 👍

**Pull request merged**